### ISSUE TITLE
Accessibility option - plain text editor

### ISSUE THREAD

**Issue by alexwarren** [textadventures/squiffy-editor#7]
Ace editor doesn't work well with screen readers, so we should have a normal textarea option.

To make this work with the section and passage dropdown, see [this](http://stackoverflow.com/questions/9185630/find-out-the-line-row-number-of-the-cursor-in-a-textarea).
